src/ripple/rpc/impl/ShardArchiveHandler.cpp

```diff
@@ -391,5 +581,42 @@ ShardArchiveHandler::remove(std::lock_guard<std::mutex>&)
     }
 }
 
+void
+ShardArchiveHandler::doRelease(std::lock_guard<std::mutex> const&)
+{
+    process_ = false;
+
+    timer_.cancel();
+    for (auto const& ar : archives_)
+        app_.getShardStore()->removePreShard(ar.first);
+    archives_.clear();
+
+    {
+        auto db = conn_->checkoutDb();
+
+        *db << "DROP TABLE State;";
+        *db << "DROP TABLE MetaState;";
+    }
+
+    conn_.reset();
+
+    // Remove temp root download directory
+    try
+    {
+        remove_all(downloadDir_);
+        remove_all(stateDir_);
+    }
+    catch (std::exception const& e)
+    {
+        JLOG(j_.error()) <<
+                         "exception: " << e.what();
```

Comment:
Incorrect indentation

Reply:
Consider logging where the exception was caught.
eg.
```
JLOG(j_.error()) <<
    "exception " << e.what() << " in function " << __func__;
```

Reply:
Done